internal/kafka/internal/presenters/kafka.go

```diff
@@ -85,7 +86,7 @@ func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 		CloudProvider:              kafkaRequest.CloudProvider,
 		MultiAz:                    kafkaRequest.MultiAZ,
 		Owner:                      kafkaRequest.Owner,
-		BootstrapServerHost:        setBootstrapServerHost(kafkaRequest.BootstrapServerHost),
+		BootstrapServerHost:        setBootstrapServerHost(kafkaRequest),
 		AdminApiServerUrl:          kafkaRequest.AdminApiServerURL,
```

Comment:
The JIRA also mentions stop showing `AdminApiServerUrl` depending on the suspending/suspended/resuming state.

Reply:
Thanks @machi1990 . Just added the function to set the Url.